go.mod

```diff
@@ -31,6 +31,21 @@ require (
 	google.golang.org/protobuf v1.28.0
 )
 
+require (
```

Comment:
Nit: can we combine this require block with the one below?

Reply:
Fixed! I think this issue actually comes from merge conflicts being resolved slightly wrong. If we resolve a merge conflict but retain multiple require blocks, then we find oursevles in a situation where we have 2+ require blocks in `go.mod`.

Something for us to keep an eye on in the future.